admin/release.py

```diff
@@ -460,16 +482,110 @@ def upload_rpms(scratch_directory, target_bucket, version, build_server):
         )
 
 
-def publish_rpms_main(args, base_path, top_level):
+packages_template = (
```

Comment:
:question: Is this nicer than https://pypi.python.org/pypi/html/, apart from not introducing another dependency?